python/tox.ini

```diff
@@ -28,14 +38,13 @@ commands =
     rst-lint --encoding 'utf-8' README.rst
 
 [testenv:doc]
-usedevelop = True
 deps =
-    sphinx
+    sphinx==1.8.5
```

Comment:
What's the reason to lock this version?

Reply:
sphinx released 2.0 onthe day i was sinking the many hours into build stuff, and it somehow breaks doc-generation.  1.8.5 works so i just fixated it.  time to worry about this for another day but we are not making heavy use of sphinx so no biggie i think.